src/fortran/cantera_thermo.f90

```diff
@@ -217,6 +217,13 @@ subroutine ctthermo_setMassFractionsByName(self, y)
       self%err = phase_setmassfractionsbyname(self%thermo_id, y)
     end subroutine ctthermo_setmassfractionsbyname
 
+    subroutine ctthermo_setMassFractions_NoNorm(self, x)
+      implicit none
+      type(phase_t), intent(inout) :: self
+      double precision, intent(in) :: x(self%nsp)
+      self%err = phase_setMassFractions(self%thermo_id, x, 0)
+    end subroutine ctthermo_setMassFractions_NoNorm
```

Comment:
Is there a reason to introduce this function instead of just calling `ctthermo_setMassFractions` with the third argument set to 0?